.craft.yml

```diff
@@ -6,101 +6,104 @@ targets:
   ## 1. Base Packages, node or browser SDKs depend on
   ## 1.1 Types
   - name: npm
-    id: "@sentry/types"
+    id: '@sentry/types'
```

Comment:
Prettier changed this for me 😅 Hope that's OK...

Reply:
Ahh yes, when I worked on this, my prettier/vscode settings were broken and apparently our lint check didn't catch it. Should be fine.